pnpbridge/src/adapters/src/Camera/CameraIotPnpDeviceAdapter.h

```diff
@@ -3,59 +3,66 @@
 
 #include <string>
 
-#include <digitaltwin_device_client.h>
-#include <digitaltwin_interface_client.h>
-
-// Represents a PnP interface. It will register for its respective digital twin and then watch for
-// camera arrivals and removals. When the camera device arrives, it will create a CameraIotPnpDevice
-// for it and start sending telemetry if Start() has been called. When the camera device is removed
-// the CameraIotPnpDevice instance is removed.
+// Represents a PnP interface. It will = watch for camera arrivals and removals. When the camera 
+// device arrives, it will create a CameraIotPnpDevice for it and start sending telemetry if Start()
+// has been called. When the camera device is removed the CameraIotPnpDevice instance is removed.
 class CameraIotPnpDeviceAdapter
 {
 public:
     static std::unique_ptr<CameraIotPnpDeviceAdapter> MakeUnique(
-        const std::string& interfaceId,
         const std::string& componentName,
-        const std::string& cameraId,
-        DIGITALTWIN_INTERFACE_CLIENT_HANDLE* interfaceClient);
+        const std::string& cameraId);
 
     CameraIotPnpDeviceAdapter(
-        const std::string& cameraId);
+        const std::string& cameraId,
+        const std::string& componentName);
 
-    void Initialize(
-        DIGITALTWIN_INTERFACE_CLIENT_HANDLE interfaceClientHandle);
+    void Initialize();
 
     // Allows properties, telemetry, and to be sent
     HRESULT Start();
 
     // Stops any properties and telemetry from being sent
     void Stop();
 
-private:
-    static void CameraPnpCallback_ProcessCommandUpdate(
-        const DIGITALTWIN_CLIENT_COMMAND_REQUEST* dtCommandRequest,
-        DIGITALTWIN_CLIENT_COMMAND_RESPONSE* dtCommandResponse,
-        void* userInterfaceContext);
+    void SetIotHubDeviceClientHandle(IOTHUB_DEVICE_CLIENT_HANDLE DeviceClientHandle);
+
+    static int CameraPnpCallback_ProcessCommandUpdate(
+        _In_ PNPBRIDGE_COMPONENT_HANDLE PnpComponentHandle,

```

Comment:
Should the SAL annotations be removed? It doesn't look other functions in this file (or other files in general) use them

Reply:
Removing them for consistency.

---
In reply to: [470752600](https://github.com/Azure/AzurePnPBridgePreview/pull/34#discussion_r470752600) [](ancestors = 470752600)